### ISSUE TITLE
Windows 7: Husky fails because node can't find node in mode_modules?

### ISSUE THREAD

**Issue by RefinedSoftwareLLC** [typicode/husky#436]
Windows 7 64-bit.
node v11.6.0
npm 6.5.0
husky@1.2.1
git version 2.20.1.windows.1
Pushing to GitLab.
Only when Husky is installed into my project, do I get the following error message:

Commit failed - exit code 1 received, with output: '/c/Program Files/nodejs/node: line 1: /node_modules/node/bin/node: No such file or directory'

**Comment by typicode** [comment]
Hi @RefinedSoftwareLLC,

Not really sure what's happening.

Are you using a GUI client? Can you try running with `HUSKY_DEBUG=true` and paste the output?

I would also try installing husky in a new directory to see if it's related to the Node installation or this particular project.

In a new directory, run:

```
git init
npm init -y
npm install husky --save-dev
```

Edit your `package.json`

```
{
  "husky": {
    "hooks": {
      "pre-commit": "echo \"hello world\" && exit 1"
    }
  }
}
```

Then try to commit, you should see `hello world`  and the commit should fail.

**Comment by RefinedSoftwareLLC** [comment]
This might be related (git's linux commands don't convert "node" to "node.exe"):
```
C:\>which node
which: no node in (...myPath...)

C:\>which node.exe
/c/Program Files/nodejs/node.exe
```

I changed "./git/hooks/pre-commit" line:
```
node "$scriptPath" $hookName "$gitParams"
```
to:
```
# node.exe "$scriptPath" $hookName "$gitParams"
```
then ran again:
```
C:\myPath>set HUSKY_DEBUG=true
C:\myPath>git commit -m "myMsg"
husky:debug pre-commit hook started...
husky > pre-commit (node v11.6.0)
...myPreCommitOutput...
husky:debug prepare-commit-msg hook started...
/c/Program Files/nodejs/node: line 1: /node_modules/node/bin/node: No such file or directory
```

**Comment by localhorst** [comment]
I have experienced the same problem on Windows 10.

Windows 10 64-bit.
node 8.12.0
nvm for Windows. 1.1.7.
npm 6.4.1
husky@1.2.1
git 2.24.1.windows.2
Pushing to GitLab.

/c/Program Files/nodejs/node: line 1: /node_modules/node/bin/node: No such file or directory

When I removed the "node" file in c:\Program Files\nodejs and replaced it by a copy of node.exe, everything worked. 

Question: 
Is this a problem with husky or with Node / nvm for Windows?
